docs/configuration.md

```diff
@@ -477,6 +477,64 @@ Apart from these, the following properties are also available, and may be useful
     <code>spark.io.compression.codec</code>.
   </td>
 </tr>
+<tr>
+  <td><code>spark.shuffle.chimera.crypto.cipher.suite</code></td>
+  <td>AES/CTR/NoPadding</td>
+  <td>
+    Cipher suite for crypto codec.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.shuffle.chimera.crypto.codec.classes</code></td>
+  <td></td>
+  <td>
+    The prefix for a given crypto codec, contains a comma-separated
```

Comment:
"Comma-separated list of crypto codec implementations to use for shuffle file encryption. The first available implementation will be used."